app/components/homePageModule/HomePageController.js

```diff
@@ -3,10 +3,12 @@
 // Controller naming conventions should start with an uppercase letter
 // function HomePageController($scope, $log, $http, TestFactory1, disclosureApi) {
 function HomePageController($scope, $log, $http, TestFactory1) {
+  var localitiesList;
+
   $scope.searchBarEnabled = false;
   $scope.testVar = 'We are up and running using a required module!';
   //$scope.searchResults = [];
-  var localitiesList = {};
+  localitiesList = {};
```

Comment:
@KyleW you can pull this variable.  The controllers aren't sharing scopes so if it's not being used in the controller where it's specified, it's not being used at all.